releasenotes/notes/update-plot-gate-map-9ed6ad5490bafbbf.yaml

```diff
@@ -0,0 +1,26 @@
+---
+features:
+  - |
+    The :func:`~.plot_gate_map` visualization function and the functions built
+    on top of it, :func:`~.plot_error_map` and :func:`~.plot_circuit_layout`,
+    have a new keyword argument, ``qubit_coordinates``. This argument takes
+    a sequence of 2D coordinates to use for plotting each qubit in the backend
+    being visualized. If specified this sequence must have a length equal to
```

Comment:
The doc strings don't seem to mention the new parameter being optional. Is it worth documenting in corresponding methods?

Reply:
I kind of just assumed people would see it was a kwarg with a default of None they'd know it was optional. But it's good to be explicit in the docstring, I updated it here: https://github.com/Qiskit/qiskit-terra/pull/7814/commits/5b2d5ecb2ee759dcb7b99d4df83f1e5333bb80ec